src/outlineView.ts

```diff
@@ -313,26 +241,23 @@ function getIcon(iconType?: string, kindType?: string) {
     kindType = iconType
   }
 
-  let type: string | undefined
+  let type: string = "•"
   if (typeof kindType === "string" && kindType.length > 0) {
     let kindClass: string
     // hasKind
     if (kindType.indexOf("type-") === 0) {
       // supplied with type-...
       kindClass = `${kindType}`
       type = kindType.replace("type-", "")
-    } else if (supportedTypes.includes(kindType)) {
+    } else {
       // supplied without type-
       kindClass = `type-${kindType}`
       type = kindType
-    } else {
-      // as is
-      kindClass = kindType
```

Comment:
I think keeping this would allow the clients to use custom icons. I don't know any client that does this, but there may be one out there.

Reply:
But clients can still use custom icons, they just need to add `type-something` class instead of just `something` class. I don't think this behaviour is described as public API anywhere, so it should be (arguably) fine to change it, but I can revert this. The issue is, this requires keeping a list of known classes in js in addition to less, which irks me quite a bit (you know, DRY and all that)

Reply:
What about keeping it like this and wait for the clients to complain if they were using a custom icon?

Reply:
I'm generally not a huge fan of this approach, but I don't think there are many clients to begin with, let alone those using an undocumented feature, so I'm fine with releasing as is and waiting for complaints.

Reply:
OK. Agreed.